src/pageObject.js

```diff
@@ -1,16 +1,41 @@
 'use strict';
 
 class Header {
-
-};
+  get logo() {
+    return 'Conduit logo';
+  }
+}
 
 class PageObject {
+  constructor(url) {
+    this.url = url;
+    this.header = new Header();
+  }
```

Comment:
In the constructor of the `PageObject` class, you're creating a new instance of the `Header` class. This could lead to issues if the `Header` class ever needs to be instantiated with different parameters. Consider passing an instance of `Header` into the `PageObject` constructor instead of hardcoding it here.